api/services/microvm/v1alpha1/microvms.proto

```diff
@@ -58,11 +58,13 @@ message CreateMicroVMResponse {
 message DeleteMicroVMRequest {
   string id = 1;
   string namespace = 2;
+  string uuid = 3;
```

Comment:
I'm not sure about this. I think we need 2 versions of the Delete/Get API methods:
- Delete/Get(name, namespace)
- DeleteById/GetById(id)

I still think its valid to get or delete  a microvm based on its namespace/name

Reply:
This would probably then mean changing `id` to `name` throughout flintlock

Reply:
I wouldn't let users to delete by `ns/name` only because in some cases it will not be only one, same for get. The get endpoint returns one spec, but without the uid, it's not guaranteed that's the only one.

Reply:
Or we have to update the Get return a list of specs.

Reply:
I would even go remove everything from the delete and get and leave it only the uid. So you can Get (single spec response) only with uid and delete the same way.

Reply:
Yes i think perhaps you are right. Get and delete should only accept the id.  I think we should also implement (in a different PR) a `Find` that we can use to find by the ns/name.....wdyt?